syntax = "proto3";

message FirstLogin {
  string login = 1;
  string password = 2;
}
